### DISCUSSION TITLE
How to use LEDS ?

### DISCUSSION THREAD

**Discussion by jlr91** [maartentamboer/X-Touch-Mini-FS2020#125] [Q&A]
Hi, I've made some configurations for C152X, C172 legacy and C172 G1000. But I've never succeded in playing with leds.
I' tried a lot of things but no one worked.
Can somebody give me some complex examples such as using a mobiflight preset.
I tried to use these :
"simvar_led":"(L:C152X_DOOR_PILOT,Number)"
"mobiflightsimvar_led":"Mobiflight.C152X-window-pilot-handle-toggle"
and some others but nothing worked

**Comment by maartentamboer** [comment]
In this file there is plenty of uses for the mobiflightsimvar_led https://github.com/maartentamboer/X-Touch-Mini-FS2020/blob/main/Configurations/config_crj.json 

But for the leds to work the application is very basic (This applies to both regular simvars and mobiflight ones): https://github.com/maartentamboer/X-Touch-Mini-FS2020/blob/main/pushbutton.py#L89-L99 It comes down to when the simvar is 1 the led turns on and anything else it will be off

For more advanced usage you will need to use jinja2 scripts with Triggers. Do a search for https://github.com/maartentamboer/X-Touch-Mini-FS2020/search?q=set_button_led and take a look at https://dev-tty.nl/X-Touch-Mini-FS2020/configuration/advanced/conditional-events/

**Comment by maartentamboer** [comment]
Also, do you have mobiflight and the wasm module installed?
https://www.mobiflight.com/en/index.html

**Comment by jlr91** [comment]
Thanks for
